packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -156,16 +156,17 @@ const NotificationDialog = (props) => {
   /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
   return (
     <AbstractModal
-      ariaLabel={signalWord}
-      aria-labelledby="notification-dialog-signal-word"
-      aria-describedby={dialogTitle ? 'notification-dialog-title' : 'notification-dialog-signal-word'}
-      role="alertdialog"
+      ariaLabel=""
```

Comment:
Why is this needed? Why not remove the prop?

Reply:
`ariaLabel` is marked as required in AbstractModal. It gives a console error if not passed. So passing an empty string prevents that error and the screen reader announcements are taken care of by other aria attributes